spec/components/candidate_interface/course_choices_review_component_spec.rb

```diff
@@ -140,6 +139,36 @@
       end
     end
 
+    context 'When a candidate has a missing gcse' do
+      let(:application_choice) { application_form.application_choices.first }
+      let(:application_qualification) { create(:gcse_qualification, :missing, currently_completing_qualification: false) }
```

Comment:
this is going to create an application form. You can use build instead of create 👍